exporters/metrics/src/main/java/com/google/cloud/opentelemetry/metric/MetricTranslator.java

```diff
@@ -54,6 +61,17 @@
   static final Set<MetricDataType> DOUBLE_TYPES = ImmutableSet.of(DOUBLE_GAUGE, DOUBLE_SUM);
   private static final int MIN_TIMESTAMP_INTERVAL_NANOS = 1000000;
 
+  // Mapping outlined at https://cloud.google.com/monitoring/api/resources#tag_gce_instance
+  private static final Map<String, AttributeKey<String>> gceMap =
+      Stream.of(
```

Comment:
I find readability of this code questionable (we could just write a static method which would return a map initialized via multiple `.put(...)` calls instead), but it's fine to improve that in a follow-up PR if needed.